prescriptions-refresh-job/base/argo-workflows/prescriptions-refresh-image-analysis.yaml

```
@@ -1,110 +1,167 @@
-          - name: GITHUB_PRIVATE_KEY_PATH
-            value: /opt/app-root/src/.github/github-privatekey
-          - name: THOTH_LOGGING_NO_JSON
-            valueFrom:
-              configMapKeyRef:
-                name: thoth
-                key: logging-no-json
-          - name: THOTH_PRESCRIPTIONS_REFRESH_GITHUB_TOKEN
-            valueFrom:
-              secretKeyRef:
-                name: kebechet
-                key: github-oauth-token
-          - name: THOTH_PRESCRIPTIONS_REFRESH_QUAY_TOKEN
-            valueFrom:
-              secretKeyRef:
-                name: thoth
-                key: quay-token
-          - name: GITHUB_APP_ID
-            valueFrom:
-              secretKeyRef:
-                name: kebechet
-                key: GITHUB_APP_ID
-          - name: THOTH_PRESCRIPTIONS_REFRESH_GITHUB_LABELS
-            value: bot
-          - name: THOTH_PRESCRIPTIONS_REFRESH_REPO
-            value: git@github.com:thoth-station/prescriptions.git
-          - name: THOTH_PRESCRIPTIONS_REFRESH_DEBUG
-            value: "1"
-          - name: PROMETHEUS_PUSHGATEWAY_HOST
-            valueFrom:
-              configMapKeyRef:
-                key: pushgateway-host
-                name: prometheus
-          - name: PROMETHEUS_PUSHGATEWAY_PORT
-            valueFrom:
-              configMapKeyRef:
-                key: pushgateway-port
-                name: prometheus
-        resources:
-          limits:
-            cpu: 250m
-            memory: 256Mi
-          requests:
-            cpu: 250m
-            memory: 256Mi
-        volumeMounts:
-          - name: ssh-config
-            mountPath: /opt/app-root/src/.ssh
-          - name: github-app-privatekey
-            mountPath: /opt/app-root/src/.github/
-            readOnly: true
+        - name: duration_seconds_histogram
+          help: Duration of workflow when succeeded
+          when: "{{workflow.status}} == Succeeded"
+          labels:
+            - key: name
+              value: prescriptions-refresh-image-analysis
+          histogram:
+            buckets:
+              - 5
+              - 10
+              - 30
+              - 60
+              - 120
+              - 180
+              - 300
+              - 600
+              - 900
+            value: "{{workflow.duration}}"
+    volumes:
+      - name: ssh-config
+        secret:
+          secretName: kebechet
+          items:
+            - key: ssh-privatekey
+              path: id_rsa
+              mode: 0600
+      - name: github-app-privatekey
+        secret:
+          secretName: kebechet
+          items:
+          - key: KEBBHUT_GITHUB_PRIVATE_KEY
+            path: github-privatekey
+    entrypoint: image-analysis
+    templates:
+      - name: image-analysis
+        resubmitPendingPods: true
+        container:
+          name: image-analysis
+          image: prescriptions-refresh-job
+          env:
+            - name: THOTH_PRESCRIPTIONS_DRY_RUN
+              value: 1
```

```suggestion
              value: "1"
```

int values are not accepted.

---

Fixed in https://github.com/thoth-station/thoth-application/pull/2312/commits/eee94e2dc151a5a32ecf5b897ca49259e99ff63b